File: SocratesFr/CountCoversByDiet/MealChoice.cs
using System;

namespace SocratesFr.CountCoversByDiet
{
    public class MealChoice
    {
        public MealChoice()
        {
        }
        
        public int CountCovers(MealTime mealTime, Diet diet)
        {
            return 1;
        }
    }

    public enum MealTime
    {
        FridayLunch,
        SundayLunch,
    }

    public enum Diet
    {
        Vegan,
    }
}

File: src/main/java/fr/lacombe/socratesfr/MealChoice.java
package fr.lacombe.socratesfr;

class MealChoice {

    private final Meal meal;
    private final Diet diet;

    public MealChoice(Meal meal, Diet diet) {
        this.meal = meal;
        this.diet = diet;
    }

    public Diet getDiet() {
        return diet;
    }

    public Meal getMeal() {
        return meal;
    }
}
